modules/nf-core/bcftools/reheader/meta.yml

```
@@ -44,8 +44,8 @@ output:
       pattern: "versions.yml"
   - vcf:
       type: file
-      description: VCF with updated header
-      pattern: "*.{vcf.gz}"
+      description: VCF with updated header, bgzipped per default
+      pattern: "*.{vcf,vcf.gz,bcf,bcf.gz}"
 
 authors:
   - "@bjohnnyd"
```

```suggestion
  - "@bjohnnyd"
  - "@jemten"
```

You can add yourself as an author if you want :)